modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -77,10 +77,10 @@ the associated capability.`),
 	return cmd
 }
 
-func newSubmitTxCmd() *cobra.Command {
+func newSendTxCmd() *cobra.Command {
```

Comment:
Sorry, this is probably a bit nitpicking, but I was just thinking that what we send to the host chain is a message, right? Or is message and transaction something that mean the same thing? For me I thought that a transaction was the execution of the message, but maybe I am just mistaken... Saying this because maybe this could be renamed to `SendMsg`...

Reply:
I think we should just stick with the naming used in the spec. I see your reasoning but the `InterchainAccountPacketData` type may be composed of one or many `sdk.Msg`s encoded into the `[]byte` field.